tests/test_coordinates/test_astronomical_quantities.py

```diff
@@ -0,0 +1,207 @@
+        julian = Quantity(2454545, u.jd2000)
+        actual_era = earth_rotation_angle(julian).data
+        expected_era = 6.2360075 * 180 / np.pi
+
+        self.assertAlmostEqual(expected_era, actual_era, delta=0.01)
+
+    def test_nutation_angles_for_validation(self):
+        """
+        Notes
+        -----
+        Test case is taken from Astronomical Algorithms.[1]_
+
+        References
+        ----------
+        .. [1] Jean Meeus. Astronomical algorithms. 2nd ed. Willmann-Bell,
+           1998, pp. 148. isbn: 9780943396613.
+        """
+        julian = Quantity(2446895.5, u.jd2000)
+        d, m, n, f, o = nutation_angles(julian)
+
+        self.assertAlmostEqual(d.data, 136.9623, places=4)
+        self.assertAlmostEqual(m.data, 94.9792, places=4)
+        self.assertAlmostEqual(n.data, 229.2784, places=4)
+        self.assertAlmostEqual(f.data, 143.4079, places=4)
+        self.assertAlmostEqual(o.data, 11.2531, places=4)
+
+    def test_nutation_components_for_validation(self):
+        """
+        Notes
+        -----
+        Test cases are taken from the PHP Science Labs.[1]_
+
+        References
+        ----------
+        .. [1] Jay Tanner. NeoProgrammics - Science Computations. 2021.
+           url: http://www.neoprogrammics.com/nutations/.
+        """
+        julian = Quantity(2449634.50000, u.jd2000)
+
+        expected_nutation_in_longitude = 11.694
+        expected_nutation_in_obliquity = -5.946
+        actual_nutation_in_longitude, actual_nutation_in_obliquity = \
+            nutation_components(julian)
+
+        self.assertAlmostEqual(expected_nutation_in_longitude,
+                               actual_nutation_in_longitude.data, delta=0.5)
+        self.assertAlmostEqual(expected_nutation_in_obliquity,
+                               actual_nutation_in_obliquity.data, delta=0.1)
+
+    def test_mean_obliquity_for_validation(self):
+        """
+        Notes
+        -----
+        Test cases are taken from PHP Science Labs.[1]_
+
+        References
+        ----------
+        .. [1] Jay Tanner. Obliquity of the Ecliptic - PHP Science Labs. 2021.
+           url: https://www.neoprogrammics.com/obliquity_of_the_ecliptic/Obliq
+           uity_Of_The_Ecliptic_Calculator.php
+        """
+        julian = Quantity(2459437.81600, u.jd2000)
+
+        expected_mean_obliquity = 23.43648
+        actual_mean_obliquity = mean_obliquity(julian).data
+
+        self.assertAlmostEqual(expected_mean_obliquity, actual_mean_obliquity,
+                               delta=0.0001)
+
+    def test_apparent_obliquity_for_validation(self):
+        """
+        Notes
+        -----
+        Test cases are taken from PHP Science Labs.[1]_
+
+        References
+        ----------
+        .. [1] Jay Tanner. Obliquity of the Ecliptic - PHP Science Labs. 2021.
+           url: https://www.neoprogrammics.com/obliquity_of_the_ecliptic/Obliq
+           uity_Of_The_Ecliptic_Calculator.php
+        """
```

Comment:
Add blank lines between the doc strings and code. Do for all methods in this file.